### PR TITLE
[feat] allow revocation of all users or activations

### PR DESCRIPTION
**Pull request by aricart**
added the ability to revoke users/activations by timestamp. If the value for the public key is "*", any JWT issued before the timestamp specified will be considered revoked.

### CODE REVIEW

**Review by matthiashanel — APPROVED** [review]
Except for comment and go.mod stuff LGTM

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,7 @@
 module github.com/nats-io/jwt
 
 require (
+	github.com/nats-io/jwt/v2 v2.0.0-20201015190852-e11ce317263c
```
[the remark this answers is not in the document]
Comment (aricart) [reply go.mod]: Good catch the scripts were not fmt or running tests for both versions on local env.

File: v2/go.sum [hunk 2]
```diff
@@ -1,3 +1,4 @@
+github.com/nats-io/jwt/v2 v2.0.0-20201015190852-e11ce317263c/go.mod h1:vs+ZEjP+XKy8szkBmQwCB7RjYdIlMaPsFPs4VdS4bTQ=
```
[the remark this answers is not in the document]
Comment (aricart) [reply v2/go.sum]: go mod tidy fixed all that

**Review by matthiashanel — APPROVED** [review]
LGTM

File: revocation_list.go [hunk 3]
```diff
@@ -39,9 +41,19 @@ func (r RevocationList) ClearRevocation(pubKey string) {
 }
 
 // IsRevoked checks if the public key is in the revoked list with a timestamp later than
-// the one passed in. Generally this method is called with time.Now() but other time's can
+// the one passed in. Generally this method is called with an issue time but other time's can
```
Comment (kozlovic) [inline revocation_list.go:44]: Sorry, I just merged #113 that was a port of some changes that were made in v2/ but not v1/ but were done in the branch_v1 :-). So you should rebase and see if there are some changes that are no longer needed.

**Pull request merged**